src/date-utils.js

```diff
@@ -1,4 +1,8 @@
-import { format } from 'date-fns'
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+
+// dayjs plugins
+dayjs.extend(relativeTime)
 
 const KNOWN_FORMATS = {
   iso: "yyyy-MM-dd'T'HH:mm:ss",
```

Comment:
Let's remove both `iso` and `standard` (I don't think we use them at all); day.js applies `iso` by default it seems: https://day.js.org/docs/en/display/format